doc/elements.md

```diff
@@ -187,6 +187,32 @@ A = np.array([[1.23], [4.56]])
 If a variable `v` is a complex object, you should use `import prairielearn as pl` and `data['params'][params_name] = pl.to_json(v)`.
 
 
+## `pl-latex-matrix-display` element
+
+```html
+<pl-latex-matrix-display params-name="A"></pl-latex-matrix-display>
+```
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`params-name` | string | — | Name of variable in `data['params']` to display.
+`label` | text | — | A prefix to display before the matrix (e.g., `label="${\bf A} = $"`).
+`presentation-type` | string | `'f'` | Number display format. If `presentation-type` is `'sigfig'`, each number is formatted using the `to_precision` module to `digits` significant figures.  Otherwise, each number is formatted as `'{:.{digits}{presentation-type}}'`.
+`digits` | integer | `"2"` | Number of digits to display according to the choice of `presentation-type`
+
+The variable in `data['params']` must be a 1D or 2D numpy array. For example, if we want to display the following matrix operations
+```
+x = [A][b] + [c]
+```
+we write
+
+```html
+<pl-latex-matrix-display params-name="A"  digits="1" label="${\bf x} = $"></pl-latex-matrix-display>
+<pl-latex-matrix-display params-name="b"  digits="1"></pl-latex-matrix-display>
```

Comment:
I don't remember - is it possible to use this syntax instead?
`<pl-latex-matrix-display params-name="b"  digits="1" />`

Reply:
As far as I know, we should use the syntax that I implemented, where all tags are explicitly closed.

Reply:
yep - self-closing tags that aren't explicitly self-closing in the HTML spec will fail under an upcoming parser change.